cypress/integration/pages/frontPage/testsForCanonicalOnly.js

```diff
@@ -1,5 +1,10 @@
 // For testing important features that differ between services, e.g. Timestamps.
 // We recommend using inline conditional logic to limit tests to services which differ.
+import toggles from '../../../../src/app/lib/config/toggles';
```

Comment:
We have a helper for toggles in https://github.com/bbc/simorgh/blob/latest/cypress/support/helpers/useAppToggles.js.
Might be better to use that instead

Reply:
Done.